integration-tests/src/test/java/org/zowe/apiml/functional/gateway/GatewayAuthenticationTest.java

```diff
@@ -42,14 +42,16 @@ void setUp() {
     class GivenBearerAuthentication {
         @Nested
         class WhenAccessingProtectedEndpoint {
```

Comment:
Should there be a test when `${apiml.gateway.health.protected:false}")` is false by default?

Reply:
I have added a new test that will run as a part of Zaas tests - I didn't find a better "group" but I am open to suggestions